### DISCUSSION TITLE
SmartAutoClicker-2.3.1

### DISCUSSION THREAD

**Discussion by Nain57** [Nain57/Smart-AutoClicker#314] [Announcements]
Fix loop crash due to a migration issue.
See 2.3.0 change note for all details of the update: https://github.com/Nain57/Smart-AutoClicker/releases/tag/2.3.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Nain57/Smart-AutoClicker/releases/tag/2.3.1'>SmartAutoClicker-2.3.1</a>.</em>

**Comment by DrankRock** [comment]
I bought the premium version using the play store, is their a way to transfer it to the github apk ? 
Thanks !

**Reply by Nain57** [reply]
GitHub/F-Droid apk are completely free, there is no "pro mode" on it, everything is available for free

**Reply by Nain57** [reply]
Just to let you know, PlayStore version will be updated, but Google feel like they need to verify my identity (for like the fifth time in two months).
I can't push an update until they review my passeport again 😅

**Reply by DrankRock** [reply]
Good lord, you are the savior of my life *-*, i will use this apk then :)
Is there somewhere i can give you money then ? a Ko-fee or what ?

**Reply by stansu** [reply]
i also want to know how to sponsor the author.

**Comment by c-sanchez** [comment]
It would be great if you add options to make donations in the project page, as Paypal, Payeer, Binance Pay, UDST TRC20, Bitcoin.
